src/main/resources/templates/fragments/fileUploader.html

```diff
@@ -73,6 +73,8 @@
     var maxFileSize = Number(
         [[${@environment.getProperty('form-flow.uploads.max-file-size')}]]);
 
+    mixpanel.init('[[${@environment.getProperty("MIXPANEL_API_KEY")}]]');
```

Comment:
I dont think this is needed here? I think it's already included in the head/mixPanel fragment